os/arch/xtensa/src/esp32/esp32_spi.c

```diff
@@ -159,26 +159,26 @@ typedef struct {
 
 typedef union {
 	struct {
-		uint32_t rd_buf_done: 1;	/*The interrupt raw bit for the completion of read-buffer operation in the slave mode. */
-		uint32_t wr_buf_done: 1;	/*The interrupt raw bit for the completion of write-buffer operation in the slave mode. */
-		uint32_t rd_sta_done: 1;	/*The interrupt raw bit for the completion of read-status operation in the slave mode. */
-		uint32_t wr_sta_done: 1;	/*The interrupt raw bit for the completion of write-status operation in the slave mode. */
-		uint32_t trans_done: 1;	/*The interrupt raw bit for the completion of any operation in both the master mode and the slave mode. */
-		uint32_t rd_buf_inten: 1;	/*The interrupt enable bit for the completion of read-buffer operation in the slave mode. */
-		uint32_t wr_buf_inten: 1;	/*The interrupt enable bit for the completion of write-buffer operation in the slave mode. */
-		uint32_t rd_sta_inten: 1;	/*The interrupt enable bit for the completion of read-status operation in the slave mode. */
-		uint32_t wr_sta_inten: 1;	/*The interrupt enable bit for the completion of write-status operation in the slave mode. */
-		uint32_t trans_inten: 1;	/*The interrupt enable bit for the completion of any operation in both the master mode and the slave mode. */
-		uint32_t cs_i_mode: 2;	/*In the slave mode  this bits used to synchronize the input spi cs signal and eliminate spi cs  jitter. */
-		uint32_t reserved12: 5;	/*reserved */
-		uint32_t last_command: 3;	/*In the slave mode it is the value of command. */
-		uint32_t last_state: 3;	/*In the slave mode it is the state of spi state machine. */
-		uint32_t trans_cnt: 4;	/*The operations counter in both the master mode and the slave mode. 4: read-status */
-		uint32_t cmd_define: 1;	/*1: slave mode commands are defined in SPI_SLAVE3.  0: slave mode commands are fixed as: 1: write-status 2: write-buffer and 3: read-buffer. */
-		uint32_t wr_rd_sta_en: 1;	/*write and read status enable  in the slave mode */
-		uint32_t wr_rd_buf_en: 1;	/*write and read buffer enable in the slave mode */
-		uint32_t slave_mode: 1;	/*1: slave mode 0: master mode. */
-		uint32_t sync_reset: 1;	/*Software reset enable, reset the spi clock line cs line and data lines. */
+		uint32_t rd_buf_done:1;	/*The interrupt raw bit for the completion of read-buffer operation in the slave mode. */
```

Comment:
accroding to code submiting experience,  split coding rule as separate commit.

Reply:
fixed, thanks for suggestion